go/vt/vtgate/planbuilder/insert.go

```diff
@@ -46,12 +44,12 @@ func buildInsertPlan(stmt sqlparser.Statement, reservedVars *sqlparser.ReservedV
 	// The table might have been routed to a different one.
 	ins.Table = exprs[0].(*sqlparser.AliasedTableExpr).Expr.(sqlparser.TableName)
 	if rb.eroute.TargetDestination != nil {
-		return nil, errors.New("unsupported: INSERT with a target destination")
+		return nil, vterrors.VT12001("INSERT with a target destination")
 	}
 
 	if len(pb.st.tables) != 1 {
 		// Unreachable.
-		return nil, vterrors.Errorf(vtrpcpb.Code_UNIMPLEMENTED, "multi-table insert statement in not supported in sharded keyspace")
+		return nil, vterrors.VT12001("multi-table INSERT statement is not supported in a sharded keyspace")
```

Comment:
```suggestion
		return nil, vterrors.VT12001("multi-table INSERT statement in a sharded keyspace")
```

Reply:
Since 12001 is already `unsupported` we don't need to repeat that in the message.

Reply:
resolved in [f2351bb](https://github.com/vitessio/vitess/pull/10738/commits/f2351bba65f347a8e5ce72ff87757ccaf20115af)